### PR TITLE
Updated Gradle file

### LINKED ISSUE #956: Unresolved Reference kotlinx and Updated Gradle
**Actual Behaviour**

Android Studio shows unresolved reference for kotlinx.

**Expected Behaviour**

There should be no warning for kotlin support in android studio.

**Steps to reproduce it**

Build the cloned repo

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by yashLadha**
- Fixed unresolved reference for kotlinx
- Updated gradle version for instant run
Fixes #956 

Changes: Updated gradle version and applied plugin for kotlin

Screenshots for the change: 
image

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -28,3 +28,4 @@ allprojects {
     }
 }
 
+apply plugin: 'kotlin-android-extensions'
```
Comment (collinx) [inline build.gradle]: Kotlin plugin is already applied in build.gradle (app module).

Comment (yashLadha) [reply build.gradle]: But is giving warnings when compiled. Giving unresolved reference error for kotlinx. After adding this all the warnings are gone.

Comment (collinx) [reply build.gradle]: There are no such warnings on the compilation on my side.  Do check again.

Comment (yashLadha) [reply build.gradle]: Yaa, may be it is not more, after installing the compiler.

**Pull request closed without merging**